tests/test_orbit_phase.py

```diff
@@ -20,6 +20,8 @@ def setUpClass(cls):
         os.chdir(datadir)
         cls.pJ1855 = "B1855+09_NANOGrav_dfg+12_modified_DD.par"
         cls.mJ1855 = m.get_model(cls.pJ1855)
+        cls.pJ1855ell1 = "B1855+09_NANOGrav_12yv3.wb.gls.par"
+        cls.mJ1855ell1 = m.get_model(cls.pJ1855ell1)
         cls.pJ0737 = "0737A_latest.par"
         cls.mJ0737 = m.get_model(cls.pJ0737)
         cls.timfile = "test1.tim"
```

Comment:
Is this `timfile` and the associates `toas` object ever used? Loading `.tim` files is slow and this gets executed for each test in the class.

Reply:
It is only used in one of the tests.  I guess I could move it to that particular test.

Reply:
Done.